<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleWord</class>
 <widget class="QDialog" name="SingleWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="listName">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="language1Name">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Language1</string>
   </property>
  </widget>
  <widget class="QLabel" name="language2Name">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Language2</string>
   </property>
  </widget>
  <widget class="QLabel" name="language1Word">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>lan1Word</string>
   </property>
  </widget>
  <widget class="QLabel" name="language2Word">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>lan2Word</string>
   </property>
  </widget>
  <widget class="QLabel" name="commentText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>761</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>comment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goBackButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editWordButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Edit word</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeWordButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>460</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Remove word</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
